CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 24.0.0 yyyy-mm-dd
```

Comment:
leave this as `x.x.x yyyy-mm-dd` the release script will replace it with the right version/date